src/chocolatey/infrastructure.app/configuration/EnvironmentSettings.cs

```diff
@@ -1,4 +1,4 @@
-﻿// Copyright © 2017 - 2021 Chocolatey Software, Inc
+// Copyright © 2017 - 2021 Chocolatey Software, Inc

```

Comment:
What changed here?
The removal of BOM or adding it?

If adding it, can we have that as part of a different commit?

Reply:
Hmmm... Not sure why this changed. I'll take a look and add it as it's own commit if it should change.

Reply:
Apparently Visual Studio decided this shouldn't have a BOM. Checking other cs files, it seems some have BOM, and some don't. I've added a maint commit to remove the BOM.

Reply:
We'll need to come to an agreement at some point of what files should have a BOM, and which should not.

I think my VS instance usually adds it back in (not sure), and older VS versions almost always added it in. But let us leave it at that for now.